wsgi.py

```diff
@@ -141,7 +141,7 @@ def post_publish():
         prometheus_metrics.METRICS['posts'].inc()
         _retryable(
             'post',
-            f'http://{UPLOAD_SERVICE_ENDPOINT}',
+            f'{UPLOAD_SERVICE_API}/upload',
```

Comment:
how do we decide when to expect the url `path` coming from the env and when it is not?

Reply:
@jameswnl My idea was that at services where it's possible, we would default to a scheme:
```
<SERVICE_NAME>_API == <scheme>://<host>:<port>/<path_to_api>
```
When that's not possible  we would use scheme:
```
<SERVICE_NAME>_HOST == <scheme>://<host>:<port>
<SERVICE_NAME>_PATH == <path_to_api>
```
This option is required for example when integrating with Topological inventory. Their pagination provides link to next page in a format like `/api/topological-inventory/v1/container_nodes?offset=0`, so we need to have the base host url available as well...

What do you think of that?

Reply:
Maybe the version string should be excluded from the the path to API? So in case of the upload service, that would look like:
```
UPLOAD_SERVICE_API=http://upload-service.platform-ci.svc:8080/api/ingress
```
And in code we would call `v1/upload`?

Currently it looks like this:
```
UPLOAD_SERVICE_API=http://upload-service.platform-ci.svc:8080/api/ingress/v1
```

Reply:
Either way looks good to me.

Personally, I'd like the code to have the least need to manipulate the url.  So that when anyone decide to change the schema, we can all push to the code via a change of env variable. (and obviously the paginated part is a bit tricky since different API services decide to use different terms (Host Inventory using `page=` and now Topological using `offset=`)

But yeah, I am ok with what you've proposed.  Just want an agreed on convention. that's it.